package/kernel/mt76/Makefile

```diff
@@ -243,6 +243,12 @@ define KernelPackage/mt7921-firmware
   TITLE:=MediaTek MT7921 firmware
 endef
 
+define KernelPackage/mt7922-firmware
+  $(KernelPackage/mt76-default)
+  DEPENDS+=+kmod-mt7921-common
```

Comment:
That now drags in the 7921 firmware too.
I did the dependency that way so there's no way to choose the driver without firmware.
But I guess you should change that with 7922 so you only get the firmware you want.